enterprise/developerbox/installation/board-recovery.md

```diff
@@ -26,6 +26,8 @@ In all other cases is it **strongly** recommended to use [Capsule Update](README
 ## Update using serial flasher
 
 The serial flasher can be enabled in two ways.
+Preinstalled CM3 firmware supports both ways,
+but OSS SCP-firmware only supports DSW2-7 method.
```

Comment:
Within the Developerbox documentation this kind of information is presented as a Note: (and should be a paragraph on its own). It would also be good to provide a link to show what the alternative SCP firmware is.

~~~
Note: *The pre-installed C-M3 firmware supports both flashing methods but, currently, the [upstream SCP firmware](../build/README.md#build-scp-firmware-from-source) only supports the DSW2-7 method.*
~~~